plugin/core/message_request_handler.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, view: Optional[ViewLike], client: Client, request_id: Any, pa
     def _send_user_choice(self, href: int = -1) -> None:
         if not self.request_sent and self.view:
             self.request_sent = True
-            mdpopups.hide_popup(self.view)
+            mdpopups.hide_popup(self.view)  # type: ignore
```

Comment:
There's a file `mdpopups.pyi` which provides the type hinting. If you want to fix the mypy error you should add `hide_popup` to `mdpopups.pyi` in this repo. But, you could also simply use `self.view.hide_popup()`, it does the same thing.